swagger.yaml

```diff
@@ -744,9 +804,46 @@ components:
         - MONTH
         - WEEK
 
+
+    User:
+      type: object
+      description: User Data
+      required:
+        - id
+        - firstName
+        - lastName
+        - email
+        - isActive
+      properties:
+        id:
+          type: string
+          format: uuid
+          description: Person ID in Keycloak.
+        firstName:
+          type: string
+          description: Person first name.
+        lastName:
+          type: string
+          description: Person last name.
+        email:
+          type: string
+          description: Person email.
+        isActive:
+          type: boolean
+          description: Defining is person active e.g. is employed.
+        severaGuid:
+          type: integer
+          format: int32
+          description: Persons GUID in Severa
```

Comment:
User GUID in Severa seems to be of type String [https://api.severa.visma.com/rest-api/doc/index.html#/UsersRead/Users_GetUsers](https://api.severa.visma.com/rest-api/doc/index.html#/UsersRead/Users_GetUsers)

Reply:
Resolved